.gitignore

```diff
@@ -1,2 +1,3 @@
-vendor
-composer.lock
+/vendor/
+/composer.lock
```

Comment:
To me, manala looks like a full fledged application, ready to use and not as a library your project might depend on.
After all, that's why you're building an executable phar. It represents a runnable application/tool, more than a library you'll require and use in your code.

`composer/composer` for instance (despite the fact the `composer.json` file describes it as a `library`...) locks its dependencies.